openMINDS_pipeline/resolver.py

```diff
@@ -71,12 +71,24 @@ def _schemas_by_category(schemas: List[SchemaStructure]) -> Dict[str, List[str]]
             for c in s.categories:
                 if c not in result:
                     result[c] = []
-                result[c].append(s.type)
+                # Only SANDS is in uppercase in schema_group
+                result[c].append(s.schema_group.replace("SANDS", "sands") + ':' + s.type)
```

Comment:
I think it's not ideal to handle "SANDS" specifically since we don't know if this is going to remain the only "acronym"-alike module. Question: Couldn't we rather define a generic pattern like if all letters are uppercase -> change it to all lowercase?